package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@apiaryio/metamorphoses",
-  "version": "0.13.3",
+  "version": "0.13.4",
```

Comment:
Since we also want to change how parameter types are handled (https://github.com/apiaryio/api-elements/pull/20). Should we omit the version bump here and do it once we've updated Metamorphoses for these changes too?